woocommerce/woocommerce-admin
# Remove sideloaded images, host on woocommerce.wordpress.com

Fixes #3716 

Current builds of woocommerce-admin include over 5mb of images that are ultimately sideloaded - downloaded from the plugin directory, and uploaded to the media library. These images drastically increase the size of the build and in-turn, inflate the size of WooCommerce core to over 12mb.

I attempted to run the images through a jpg optimizer, and they still weighed in at 2.7mb total - which to me feels still way too costly for what they are ultimately used for. Furthermore, any existing woo site will never use these images, since they are only leveraged during the Homepage creation step in the new site task list.

As such, the proposed solution here involves deletion of the files from the repository, and instead hosting them on `woocommerce.wordpress.com`. The sideload logic still functions the same, so when a homepage is being created, the needed images are downloaded from WPCOM, and inserted into the media library.

This does run the risk of those images potentially being deleted someday on accident, so not sure if this is the most optimal solution. I did read through the plugin repository guidelines and did not see any rules that this approach would conflict with.

/cc @peterfabian 

### Detailed test instructions:

- Disable Jetpack, otherwise the new post flow will be directed to WordPress.com and that kind of makes testing this more difficult
- Go to Orders > Help > Setup Wizard and click the Enable button for the task list:

![image](https://user-images.githubusercontent.com/22080/74476556-66fd7380-4e5e-11ea-9add-a33025925df7.png)

- Go back to the Woo dashboard, and click the Personalize Your Store step, and then the create homepage button:

![image](https://user-images.githubusercontent.com/22080/74476603-8399ab80-4e5e-11ea-8c56-16f4508760e5.png)

- Verify the image(s) used on the example layout include the images related to the industry you selected on your site, and that the images have been sideloaded/hosted from your test site's media library.

### Changelog Note:

Remove sideloaded images and host externally to save on build size
